cli/src/lib/generateManifests.js

```diff
@@ -90,6 +91,7 @@ module.exports = (paths, config, publicUrl) => {
             },
             {
                 src: 'safari-pinned-tab.svg',
+                sizes: '16x16',
```

Comment:
Fixes the Application tab crash in Chrome dev tools -- [slack thread](https://dhis2.slack.com/archives/CDX5CE15W/p1719753190061389?thread_ts=1716819630.607369&cid=CDX5CE15W)